apps/condo/domains/meter/components/AddressAndUnitInfo.tsx

```diff
@@ -0,0 +1,161 @@
+import { Property as PropertyType } from '@app/condo/schema'
+import { Col, ColProps, Form, FormInstance, Row } from 'antd'
+import { Gutter } from 'antd/lib/grid/row'
+import { DefaultOptionType } from 'antd/lib/select'
+import React, { Dispatch, SetStateAction } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Tooltip, Typography } from '@open-condo/ui'
+
+import { useValidations } from '@condo/domains/common/hooks/useValidations'
+import { MeterPageTypes, METER_TAB_TYPES } from '@condo/domains/meter/utils/clientSchema'
+import { usePropertyValidations } from '@condo/domains/property/components/BasePropertyForm/usePropertyValidations'
+import { PropertyAddressSearchInput } from '@condo/domains/property/components/PropertyAddressSearchInput'
+import { UnitInfo } from '@condo/domains/property/components/UnitInfo'
+import { PropertyFormItemTooltip } from '@condo/domains/property/PropertyFormItemTooltip'
+
+
+const FORM_ROW_MEDIUM_VERTICAL_GUTTER: [Gutter, Gutter] = [0, 20]
+const FORM_ITEM_WRAPPER_COLUMN_STYLE: ColProps = { style: { width: '100%', padding: 0 } }
+
+type PROPERTY_UNIT_INITIAL_VALUES = {
+    propertyId?: string
+    unitName?: string
+    unitType?: string
+}
+
+type ADDRESS_AND_UNIT_INFO_PROPS = {
```

Comment:
Actual code style don't allow to use screaming snake case for ts types definitions

Reply:
Please rename all occurrences to match repo code style 
```suggestion
type AddressAndUnitInfoProps = {
```